### ISSUE TITLE
Instant reload takes too much.

### ISSUE THREAD

**Issue by thruthesky** [FlutterFlow/flutterflow-issues#222]
Instant reload takes too much(long).

I understand that the build is done on the cloud and it takes some time.

But the `instant reload` should not take that much (long) like 90 seconds ( while it is stated `Reloading your app... This should take under 10 seconds.` on the screen. )

It really takes less than 20 seconds on a blank app (or on a nearly bank app).

But for some project which has more than 50 pages, it really takes more than 90 seconds. ( Not all people has fast internet, but mine is fast enough )

Here are the three videos I captured straight from my project just a while ago.

And it took 90 seconds on each instant reload. And it really like this. I do instant reload more than 100 times a day and that's all the same. It really take more than 90 seconds on each.

It would be a very nice **to put the expiry as in two hours**. Or at least to pro account users if it takes too much resources on the server.

Please, consider this as a bug not a feature request (to extend the expiry to two hours).

Thank you.

https://user-images.githubusercontent.com/5080329/207241945-bf6c87cf-284c-4428-8963-d3b91f480aba.mp4

https://user-images.githubusercontent.com/5080329/207241960-5c909466-5356-4eb4-8ade-4af0e06e2b65.mp4

https://user-images.githubusercontent.com/5080329/207241992-d0dde425-66e0-423d-9fd7-1ea99b04516a.mp4

**Comment by bulgariamitko** [comment]
I am not sure about it but there are 2 factors to this maybe 
- the more pages you have the slower 
- some times it loads faster then other depending on the server

I will consider this not as a bug, but I will bet FF team decide.

**Comment by thruthesky** [comment]
> I am not sure about it but there are 2 factors to this maybe
> 
> * the more pages you have the slower
> * some times it loads faster then other depending on the server
> 
> I will consider this not as a bug, but I will bet FF team decide.

Yes, this is not a bug. I know. I hope the expiration to be extended.

**Comment by wenkaifan0720** [comment]
Hi @thruthesky. I totally understand your situation. I'll raise a ticket to our team about this matter. Thank you! 

Since this is not really a bug, I'll close this issue.

**Comment by Franco98404** [comment]
@wenkaifan0720 any news about this? Is very frustrating the tests right now :c

**Comment by thruthesky** [comment]
Hi, @Franco98404 Please create a new post on the community site or new issue. We need to keep asking :)

**Comment by bulgariamitko** [comment]
It is absolutely normal to take more time based on the pages of the product you have. The more pages you have it will take more time to load. You don't need to open a new issue about that.

**Comment by Franco98404** [comment]
> It is absolutely normal to take more time based on the pages of the product you have. The more pages you have it will take more time to load. You don't need to open a new issue about that.

I understand but is very frustrating, for example I downloaded the code and I started to test on my computer, to load the project for the first time take about 30 seconds, doesn't have expiration time and have the hot reload that take 1 seconds

**Comment by alphanumericaracters** [comment]
i think it was y connection (Argentina: 1/2 Hamster of power) or browser (Edge) but with yours machines (Macs and worksttions) and connections... it's not.
